newsfragments/2445.feature.rst

```diff
@@ -0,0 +1 @@
+Improve gas strategy selection: Infura users now can choose between ``slow``, ``medium`` and ``fast``.
```

Comment:
Should we also mention the env variable? Or is it that it is likely to change and mentioning it would be confusing?

If we mention it this time around, later on for any subsequent change that modifies/changes its usage, we can reference the env variable and explain the modification.

Just something to think about.

Reply:
:heavy_check_mark: